vector/src/main/res/values/strings.xml

```diff
@@ -1493,17 +1494,21 @@ Why choose Riot.im?
     <string name="new_recovery_method_popup_title">New Key Backup</string>
     <string name="new_recovery_method_popup_description">A new secure message key backup has been detected.\n\nIf you didn’t set the new recovery method, an attacker may be trying to access your account. Change your account password and set a new recovery method immediately in Settings.</string>
     <string name="new_recovery_method_popup_was_me">It was me</string>
+
     <!-- Keys backup banner -->
-    <string name="keys_backup_banner_setup_line1">Never lose encrypted messages</string>
-    <string name="keys_backup_banner_setup_line2">Start using Key Backup</string>
+    <string name="keys_backup_banner_setup_line1">Secure Backup</string>
+    <string name="keys_backup_banner_setup_line2">Safeguard against losing access to encrypted messages &amp; data </string>
```

Comment:
Translation won't be update right now. You should create new strings, because the change is too important

Reply:
I did it